server/repositories/cmsQueries/__tests__/homePageQuery.spec.js

```diff
@@ -22,7 +22,9 @@ describe('Homepage query', () => {
             title: 'Lower Abs workout',
             fieldMojDescription: { summary: 'Intense lower core workout' },
             fieldMojThumbnailImage: {
-              uri: { url: 'https://cloud-platform-c3b3.eu-west-2' },
+              imageStyleUri: [
+                { tile_small: 'https://cloud-platform-c3b3.eu-west-2' },
+              ],
```

Comment:
couple of tests we could remove heree:
```
it('should show small tile image when provided', async () => {
it('should show large tile image when provided', async () => {
```

now we know that they will always exist